Luminary099/R31.agc

```diff
@@ -99,7 +99,7 @@ DOCMBASE	STORE	BASETIME	# PRECISION BASE VECTOR FOR CM
 REV83		CS	FLAGWRD7
 		MASK	AVEGFBIT
 		EXTEND
-		BZF	GETRVN		# IF AVEGFLAG SET, USE RN,VN
+		BZF	GETRVN		# IF AVEGFLAG SET, USE RN.VN
 
 		CS	FLAGWRD8
 		MASK	SURFFBIT
```

Comment:
2 lines below, no space between `SURFACE,` and `USE`

![image](https://user-images.githubusercontent.com/3440094/95020467-41c6ab00-0663-11eb-827b-226742595ce2.png)